libs/src/services/creatorNode/index.js

```diff
@@ -15,7 +15,7 @@ class CreatorNode {
    * Pulls off the secondary creator nodes from a creator node endpoint string.
    * @param {string} endpoints user.creator_node_endpoint
    */
-  static getSecondaries (endpoints) { return endpoints ? endpoints.split(',').slice(1) : [] }
+  static getSecondaries (endpoints) { return endpoints ? endpoints.split(',').slice(1).filter(Boolean) : [] }
```

Comment:
This is because the endpoint list has an empty string value at the end because of how we join the list together. This doesn't usually manifest as an error, but it did in scenario of unhealthy primary and no healthy secondaries during the rolloverNodes logic. adding this filter fixes that bug.